frontend/src/components/common/protected-route/protected-route.tsx

```diff
@@ -1,8 +1,9 @@
-import { AppRoute } from 'common/enums/enums';
```

Comment:
you don't need to change this logic here
you need to change logic in the `app.tsx`